app/views/helpers/ResultsPageHelper.scala

```diff
@@ -182,6 +197,196 @@ object ResultsPageHelper extends ViewHelper {
       throw new UnsupportedOperationException(message)
     }
 
+  def displayFullCalculationResult(
+    calculatorResult: CalculatorResult,
+    associatedCompanies: Int,
+    taxableProfit: Long,
+    distributions: Long,
+    config:Map[Int,FYConfig]
+  )(implicit messages: Messages): Html = {
+
+    def nonTabDisplay(taxDetails: Seq[TaxDetails]) = {
+      val htmlString = taxDetails.map { td =>
+        val year = td.year
+        val days = td.days
+        td.fold { flat =>
+          Seq(
+            s"""<h3 class="govuk-heading-m" style="margin-bottom: 4px;">${messages(
```

Comment:
its useful to move these to helper methods - like the ones in ViewHelper base trait